src/ui/listitem.cpp

```diff
@@ -66,4 +66,17 @@ void ListItem::onSizeHint(SizeHintEvent& ev)
   ev.setSizeHint(Size(w, h));
 }
 
+bool ListItem::onProcessMessage(Message* msgClick){
+  switch(msgClick->type()){
+    case kDoubleClickMessage:
+      if (parent())
+      {
+        return parent()->sendMessage(msgClick);
+      }else 
+        break;
+    break;
+  }
+  return Widget::onProcessMessage(msgClick);
+}
+
```

Comment:
Why is this necessary (same for the line in `listitem.h`) ? It works flawlessly even without this block of code

Reply:
to avoid bugs and conflicts